.github/workflows/ci.yml

```
@@ -0,0 +1,104 @@
+
+            <details>
+              <summary>✔️ 로그 확인하기</summary>
+              <br>
+              <a href="${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}">
+                Actions 탭에서 자세히 보기!
+              </a>
+            </details>
+
+      - name: Notify Discord Alarm
+        if: github.event_name == 'pull_request' && always()
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)